mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfiguration.java

```diff
@@ -96,7 +96,7 @@ default Time getHeartbeatInterval() {
     // ------------------------------------------------------------------------
     //  RPC related configurations
     // ------------------------------------------------------------------------
-    @Config("mantis.taskexecutor.rpc.external-address")
+    @Config({"mantis.taskexecutor.rpc.external-address", "MANTIS_TASKEXECUTOR_RPC_EXTERNAL_ADDRESS", "HOSTNAME"})
```

Comment:
remove `HOSTNAME`

Reply:
I cannot remove HOSTNAME here as I need to translate the env to the address config and I cannot set it outside the container as env (I cannot get this value from outside before starting the agent container).

Reply:
If this feels too risky I can revive the env override and fork dockerfile.